content/en/platform/corda/4.13/community/get-started/tutorials/supplementary-tutorials/tutorial-attachments.md

```diff
@@ -132,6 +132,37 @@ and which are shared explicitly. These are useful for attaching and signing audi
 that isn’t used as part of the contract logic.
 {{< /note >}}
 
+## How attachments are resolved
+
+When building a transaction, Corda automatically tries to resolve any missing class or contract by finding a suitable attachment, using the following order of priority:
+
+1. **Installed CorDapps first:** Corda searches the locally installed CorDapps directories first for a matching attachment.
+2. **Optional database fallback:** If no installed CorDapp is found, and database fallback is enabled, then Corda will search the attachment store (trusted uploaders only). To enable or disable database fallback, use the `net.corda.node.transactionbuilder.missingClassDbSearchDisabled` option; see [Attachment resolution configuration properties]({{< relref "#attachment-resolution-configuration-properties" >}}).
```

Comment:
We should specify that the database fallback is enabled by enabled...

**Optional database fallback:** If no installed CorDapp is found, and database fallback is enabled which it is by default, then Corda ...